File: aoj/itp1/topic3/3_b_print_test_cases.cc
#include <iostream>

int main() {
  int count = 0;
  while (true) {
    int x;
    std::cin >> x;
    if (x == 0) return 0;
    ++count;
    std::cout << "Case " << count << ": " << x << std::endl;
  }
}

File: aoj/itp1/topic3/3_b_print_test_cases.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    int count = 0;
    Scanner sc = new Scanner(System.in);
    while (true) {
      int x = sc.nextInt();
      if (x == 0) return;
      count++;
      System.out.println("Case " + count + ": " + x);
    }
  }
}
